api/src/test/java/org/openmrs/module/patientqueueing/api/PatientQueueingServiceTest.java

```diff
@@ -252,4 +252,36 @@ public void getMostRecentQueue_shouldReturnMostRecentPatientQueue() throws Parse
 		Assert.assertEquals(patientQueue, patientQueueingService.getMostRecentQueue(patient));
 		
 	}
+
+	@Test
+	public void savePatientQueue_shouldNotCompletePatientQueueOnEdit() throws Exception {
+
+		PatientQueueingService patientQueueingService = Context.getService(PatientQueueingService.class);
+
+		Patient patient = Context.getPatientService().getPatient(10000);
+
+		Location location = Context.getLocationService().getLocation(1);
+
+		PatientQueue patientQueue = new PatientQueue();
+		patientQueue.setPatient(patient);
+		patientQueue.setStatus(PatientQueue.Status.PENDING);
+		patientQueue.setEncounter(Context.getEncounterService().getEncounter(10000));
+		patientQueue.setLocationFrom(location);
+		patientQueue.setLocationTo(location);
+		patientQueue.setPriority(0);
+		patientQueue.setPriorityComment("Emergency");
+		patientQueueingService.assignVisitNumberForToday(patientQueue);
+		patientQueueingService.savePatientQue(patientQueue);
+
+		PatientQueue patientQueueToEdit=patientQueueingService.getPatientQueueById(patientQueue.getPatientQueueId());
+
+		patientQueueToEdit.setPriority(1);
+		patientQueueToEdit.setPriorityComment("Non-Emergency");
+
+		PatientQueue editedPatientQueue=patientQueueingService.savePatientQue(patientQueueToEdit);
+
+		Assert.assertEquals(1,editedPatientQueue.getPriority().intValue());
```

Comment:
It would look great if you put space after the comma of each parameter in a method.